src/components/Sum.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = props => (
+  <h1>
+    Sum of &nbsp;
+    {props.a}
+    &nbsp; and &nbsp;
+    {props.b}
+    &nbsp; is &nbsp;
+    {props.a + props.b}
+  </h1>
+);
```

Comment:
`props.` повторяется четыре раза, это можно избежать
```suggestion
const Sum = ({ a, b }) => (
  <h1>
    Sum of &nbsp;
    {a}
    &nbsp; and &nbsp;
    {b}
    &nbsp; is &nbsp;
    {a +b}
  </h1>
);
```